tests/infrastructure/tekton/utils.py

```diff
@@ -94,3 +106,32 @@ def process_yaml_files(file_paths, replacements, resource_kind, namespace):
         update_tekton_resources_yaml_file(file_path=file_path, replacements=replacements)
         resources.append(resource_kind(yaml_file=file_path, namespace=namespace).create())
     return resources
+
+
+def check_pipelinerun_status(
```

Comment:
You are using timeout sampler. So this is not this function should be named appropriately.